project-base/templates/Front/Inline/Cart/cartBox.html.twig

```diff
@@ -19,9 +19,9 @@
                 {% if cart is null %}
                     {{ 'Your cart is empty'|trans }}
                 {% else %}
-                    {% transchoice cart.itemsCount with { '%itemsCount%': cart.itemsCount, '%priceWithVat%': productsPrice.priceWithVat|price } %}
+                    {% trans with { '%count%': cart.itemsCount, '%itemsCount%': cart.itemsCount, '%priceWithVat%': productsPrice.priceWithVat|price } %}
```

Comment:
I am thinking, I would maybe rather rename `%itemsCount%` parameter to `%count%` to avoid the necessity of passing the same value twice, however, it would mean a change of the message ID, and hence a change in the translation files, so I am probably ok with the current solution :smile:

Reply:
resolved here: https://github.com/shopsys/shopsys/pull/2487/commits/9cff7aff70b4e77133a8567243e123ef158a9e97